tests/test-db.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+const test = require("tape-async");
+
+const DB = require("../db/DB");
+const getSha1 = require("../sha1-utils");
+
+test("getSubscriberByEmailAndToken accepts email and token and returns subscriber", async t => {
+  const testEmail = "unverifiedemail@test.com";
+  const testToken = "0e2cb147-2041-4e5b-8ca9-494e773b2cf0";
+  const subscriber = await DB.getSubscriberByEmailAndToken(testEmail, testToken);
+
+  t.ok(subscriber.email === testEmail);
+  t.ok(subscriber.verification_token === testToken);
```

Comment:
I'm not a **tape** expert, but do we want `t.ok()` or something like [`t.deepEqual()`](https://github.com/substack/tape#tdeepequalactual-expected-msg) here?

Reply:
I tend to avoid assertions on full object structures from experiences where it makes tests brittle. 😐 I'm fine using strict equality comparisons For Now™.

Reply:
Sorry, I didn't mean **full structure** comparisons, I just meant something like:

```js
t.deepEqual(subscriber.email, testEmail);
t.deepEqual(subscriber.verification_token, testToken);
```

Not really a big deal either way, I just usually find the debugging a bit better w/ `.deepEqual()` vs `.ok()`.

Using `.ok()` with `===` just tells me an expected/actual of true/false, which can sometimes be trickier to debug:

```sh
// t.ok(subscribers.length === 3);

# getSubscribersByHashes accepts hashes and only returns verified subscribers
not ok 1 should be truthy
  ---
    operator: ok
    expected: true
    actual:   false
    at: testp (/Users/pdehaan/dev/github/mozilla/blurts-server/tests/test-async.js:34:7)
    stack: |-
      Error: should be truthy
          at Test.assert [as _assert] (/Users/pdehaan/dev/github/mozilla/blurts-server/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/Users/pdehaan/dev/github/mozilla/blurts-server/node_modules/tape/lib/test.js:76:32)
          at Test.assert (/Users/pdehaan/dev/github/mozilla/blurts-server/node_modules/tape/lib/test.js:342:10)
          at Test.bound [as ok] (/Users/pdehaan/dev/github/mozilla/blurts-server/node_modules/tape/lib/test.js:76:32)
          at testp (/Users/pdehaan/dev/github/mozilla/blurts-server/tests/test-async.js:34:7)
          at <anonymous>
```

... versus `.deepEqual()` which gives a more obvious expected/actual, which makes it easier to see exactly why it failed instead of just a Boolean value:

```sh
// t.deepEqual(subscribers.length, 3);

# getSubscribersByHashes accepts hashes and only returns verified subscribers
not ok 1 should be equal
  ---
    operator: equal
    expected: 3
    actual:   2
    at: testp (/Users/pdehaan/dev/github/mozilla/blurts-server/tests/test-async.js:34:7)
    stack: |-
      Error: should be equal
          at Test.assert [as _assert] (/Users/pdehaan/dev/github/mozilla/blurts-server/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/Users/pdehaan/dev/github/mozilla/blurts-server/node_modules/tape/lib/test.js:76:32)
          at Test.equal (/Users/pdehaan/dev/github/mozilla/blurts-server/node_modules/tape/lib/test.js:384:10)
          at Test.bound [as equal] (/Users/pdehaan/dev/github/mozilla/blurts-server/node_modules/tape/lib/test.js:76:32)
          at testp (/Users/pdehaan/dev/github/mozilla/blurts-server/tests/test-async.js:34:7)
          at <anonymous>
```

But overall, I love everything. Let's just land these tests and iterate from there (assuming @nhnt11 and @lesleyjanenorton are kosher with this PR).

Reply:
Ah ... yeah that's a nicer output. Good call.